varats-core/varats/project/project_util.py

```
@@ -257,7 +258,79 @@ def wrap_paths_to_binaries(
     ])
 
 
-# ignore type as we do not have appropriate type information from benchbuild
+def copy_renamed_git_to_dest(src_dir: Path, dest_dir: Path) -> None:
+    """
+    Renames git files that were made git_storable (e.g., .gitted) back to their
+    original git name and stores the renamed copy at the destination path. The
+    original files stay untouched. Renaming and copying will be skipped if the
+    dest_dir already exists.
+
+    Args:
+        src_dir: path to the source directory
+        dest_dir: path to the destination directory
+    """
+    if os.path.isdir(dest_dir):
+        return
```

This looks like we fail silently here, maybe we should write at least a error message to the log. Providing a wrong `dest_dir` seems like a clear error/misshap to me.

---

You are rigth. I will add a log error message.